kuma/install.go

```diff
@@ -49,17 +50,25 @@ func (h *handler) installKuma(del bool) (string, error) {
 }
 
 // installSampleApp installs and creates a sample bookinfo application up and running
-func (h *handler) installSampleApp() (string, error) {
+func (h *handler) installSampleApp(name string) (string, error) {
 	return "deployed", nil
 }
 
 // installMesh installs the mesh in the cluster or the target location
-func (m *MeshInstance) installKumactl() error {
+func (m *MeshInstance) installKumactl(del bool) error {
+
 	Executable, err := exec.LookPath("./scripts/kuma/installer.sh")
```

Comment:
Could this be called `install.sh`, to align with the other scripts' names that are verbs?

Reply:
Sure yes